### PR TITLE
add security

### PR DESCRIPTION
**Pull request by WT-Liu**
Signed-off-by: Wenting Liu <liuwenting@pingcap.com>

### What problem does this PR solve?

add security policy

Problem Summary:

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

### Related changes

- PR to update `pingcap/docs`/`pingcap/docs-cn`:
- PR to update `pingcap/tidb-ansible`:
- Need to cherry-pick to the release branch

### Check List 

Tests 

- Unit test
- Integration test
- Manual test (add detailed scripts or steps below)
- No code

Side effects

- Performance regression
    - Consumes more CPU
    - Consumes more MEM
- Breaking backward compatibility

### Release note 
- No release note

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -121,11 +121,16 @@ Quick links:
...
 ## Security
 
-
 ### Security Audit
 
 A third-party security auditing was performed by Cure53. See the full report [here](./docs/Security-Audit.pdf).
 
+### Reporting Security Vulnerabilities
+
+To report a security vulnerability, please send an email to tikv-security@lists.cncf.io.
```
Comment (dcalvin) [inline README.md:130]:
```suggestion
To report a security vulnerability, please send an email to the [tikv-security](tikv-security@lists.cncf.io) group.
```

File: SECURITY.md [hunk 2]
```diff
@@ -0,0 +1,92 @@
+# TiKV Security Policy
+
+As a widely adopted distributed storage service, TiKV attaches great importance to code security. We are very grateful to users, security vulnerability researchers, etc. for reporting security vulnerabilities to us. All reported security vulnerabilities will be carefully assessed, addressed, and answered by us.
+
+This document describes the security process and policy followed by the TiKV project.
+## TiKV Security Team
```
Comment (BusyJay) [inline SECURITY.md:6]: There should be empty line surrounding headers.

File: SECURITY.md [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+## Reporting a vulnerability
+
+For all TiKV security-related defects, please send an email to tikv-security@lists.cncf.io. This mailing list is specially maintained by the TiKV security team. You will receive an acknowledgement mail within 24 hours. After that, we will give a detailed response about the subsequent process within 48 hours. Please do not submit security vulnerabilities directly as Github Issues.
+
+If you want to, you can choose to use the PGP public key provided by us to encrypt the content of the mail. The public key is on MIT ’s keyserver, and is provided at the end of this document.
+
+## Disclosure policy
+
+For known public security vulnerabilities, we will disclose the disclosure as soon as possible after receiving the report. Vulnerabilities discovered for the first time will be disclosed in accordance with the following process:
+The received security vulnerability report shall be handed over to the security team for follow-up coordination and repair work.
```
Comment (BusyJay) [inline SECURITY.md:33]: It should be bulleted list.

File: SECURITY.md [hunk 4]
```diff
@@ -0,0 +1,92 @@
...
+| 4.x   | :white_check_mark: |
+| 3.x   | :white_check_mark: |
+| 2.x   | :white_check_mark: |
+| < 2.0   | :x:                |
+
+## Reporting a vulnerability
+
+For all TiKV security-related defects, please send an email to tikv-security@lists.cncf.io. This mailing list is specially maintained by the TiKV security team. You will receive an acknowledgement mail within 24 hours. After that, we will give a detailed response about the subsequent process within 48 hours. Please do not submit security vulnerabilities directly as Github Issues.
+
+If you want to, you can choose to use the PGP public key provided by us to encrypt the content of the mail. The public key is on MIT ’s keyserver, and is provided at the end of this document.
```
Comment (BusyJay) [inline SECURITY.md:28]:
```suggestion
If you want to, you can choose to use the PGP public key provided by us to encrypt the content of the mail. The public key is provided at the end of this document.
```

**Pull request closed without merging**